lxd-agent/devlxd.go

```diff
@@ -118,7 +119,7 @@ var devlxdMetadataGet = devLxdHandler{"/1.0/meta-data", func(d *Daemon, w http.R
 	}
 
 	value := instance.Config["user.meta-data"]
-	return okResponse(fmt.Sprintf("#cloud-config\ninstance-id: %s\nlocal-hostname: %s\n%s", instance.Name, instance.Name, value), "raw")
+	return okResponse(fmt.Sprintf("#cloud-config\ninstance-id: %s\nlocal-hostname: %s\n%s", instance.CloudIinitID, instance.Name, value), "raw")
```

Comment:
is `instance-id` here a cloud-init concept or unchangeable for backwards compat?

Reply:
This is a `cloud-config` message, we don't control those fields.